pages/common/vectorize-pixelart.md

```diff
@@ -0,0 +1,8 @@
+# vectorize-pixelart
+
+> Convert PNG pixel art graphics to SVG/EPS vector images.
+> More information: <https://github.com/und3f/vectorize-pixelart>.
+
+- Convert a PNG image to a vector image format:
+
+`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|eps}}`
```

Comment:
Let's just do this for now.  Someone can come a long and discuss this for the long run later.
```suggestion
`vectorize-pixelart {{path/to/input.png}} {{path/to/output.svg|.eps}}`
```